rllib/agents/ppo/ddppo.py

```diff
@@ -41,8 +43,8 @@
 
 # Adds the following updates to the `PPOTrainer` config in
 # rllib/agents/ppo/ppo.py.
-DEFAULT_CONFIG = ppo.PPOTrainer.merge_trainer_configs(
-    ppo.DEFAULT_CONFIG,
+DEFAULT_CONFIG = PPOTrainer.merge_trainer_configs(
```

Comment:
kinda feel like merge_trainer_configs() should be a util on Trainer class?
so all these agents would just do

```
Trainer.merge_trainer_configs(
  ....
)
```

Reply:
It is defined in `Trainer` (not PPOTrainer), but since PPOTrainer is-a Trainer, it works like this, too. But yeah, we should probably call it like `Trainer.merge_trainer_configs()`.

Reply:
doe